server/server.js

```diff
@@ -1678,13 +1680,13 @@ async function afterLogin(socket, user) {
 
     await StatusPage.sendStatusPageList(io, socket);
 
+    const monitorPromises = [];
     for (let monitorID in monitorList) {
-        await sendHeartbeatList(socket, monitorID);
+        monitorPromises.push(sendHeartbeatList(socket, monitorID));
+        monitorPromises.push(Monitor.sendStats(io, monitorID, user.id));
```

Comment:
![](https://github.com/user-attachments/assets/98749b8e-2bc5-4d70-9cdd-b065d8d526e3)

Just checking this pr. Is this code correct? Because the function returns nothing.